backend/src/services/linear/api/resolvers.py

```
@@ -6709,7 +6218,85 @@ def resolve_commentCreate(obj, info, **kwargs):
         # Fields not yet supported by ORM but in GraphQL schema
         # createAsUser, createOnSyncedSlackThread, displayIconUrl,
         # doNotSubscribeToIssue, subscriberIds - these would require additional logic
+        
+        
+        target_fields = {
```

Ruff (0.14.4)

Local variable `target_fields` is assigned to but never used

Remove assignment to unused variable `target_fields`

(F841)